tests/blackbox/test_cortxcli.py

```diff
@@ -309,63 +265,46 @@ def test_2430(self):
         resp = self.iam_user_obj.create_iam_user(user_name=self.s3user_name,
                                                  password=self.s3acc_password,
                                                  confirm_password=self.s3acc_password)
-        self.log.info(
-            "Step 1: Creating a user with name %s", self.s3user_name)
+        self.log.info("Step 1: Creating a user with name %s", self.s3user_name)
         assert_true(resp[0], resp[1])
         self.log.info("Verifying user is created by listing users")
         resp = self.iam_user_obj.list_iam_user()
         self.log.info("Users list %s", resp[1])
         assert_true(resp[0], resp[1])
         self.log.info("Verified that user is created by listing users")
-        self.log.info(
-            "Step 1: Created a user with name %s", self.s3user_name)
+        self.log.info("Step 1: Created a user with name %s", self.s3user_name)
         self.log.info("Step 2: Creating access key for the user")
         resp = self.accesskeys_obj.create_s3_iam_access_key(self.s3user_name)
         assert_true(resp[0], resp[1])
         user_access_key = resp[1]["access_key"]
         user_secret_key = resp[1]["secret_key"]
         self.log.info("Step 2: Created access key for newly created user")
-        s3_user_obj = s3_test_lib.S3TestLib(
-            access_key=user_access_key,
-            secret_key=user_secret_key)
-        self.log.info(
-            "Step 3: Performing CRUD operations using valid user's credentials")
+        s3_user_obj = s3_test_lib.S3TestLib(access_key=user_access_key, secret_key=user_secret_key)
+        self.log.info("Step 3: Performing CRUD operations using valid user's credentials")
         self.log.info("Creating a bucket with name %s", self.s3bucket_name)
         resp = s3_user_obj.create_bucket(self.s3bucket_name)
         assert_true(resp[0], resp[1])
-        self.log.info(
-            "Bucket with name %s is created successfully", self.s3bucket_name)
+        self.log.info("Bucket with name %s is created successfully", self.s3bucket_name)
         create_file(self.test_file_path, count=1)
-        self.log.info(
-            "Putting object %s to bucket %s",
-            self.s3obj_name, self.s3bucket_name)
-        resp = s3_user_obj.put_object(
-            self.s3bucket_name,
-            self.s3obj_name,
-            self.test_file_path)
+        self.log.info("Putting object %s to bucket %s",
+                      self.s3obj_name, self.s3bucket_name)
```

Comment:
Fit into single line wherever possible to reduce number of lines and to avoid pylint warning ignores.

Reply:
I have fixed all issues those are possible.